doc/test-framework/scorecard.md

```diff
@@ -1,80 +1,64 @@
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+<!-- **Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)* -->
 
-## How It Works
+- [Overview](#overview)
+- [Requirements](#requirements)
+- [Running the Scorecard](#running-the-scorecard)
+- [Configuration](#configuration)
+  - [Config File](#config-file)
+  - [Command Args](#command-args)
+  - [Config File Options](#config-file-options)
+  - [Plugins](#plugins)
+    - [Basic and OLM](#basic-and-olm)
+    - [External Plugins](#external-plugins)
+- [Tests Performed](#tests-performed)
+  - [Basic Operator](#basic-operator)
+  - [OLM Integration](#olm-integration)
+- [Extending the Scorecard with Plugins](#extending-the-scorecard-with-plugins)
+  - [JSON format](#json-format)
+- [Running the scorecard with a deployed CSV](#running-the-scorecard-with-a-deployed-csv)
 
-The scorecard works by creating all resources required by CRs and the operator. For the operator
-deployment, it also adds another container to the operator's pod that is used to record calls to the API server,
-which are analyzed by the scorecard for various tests. The scorecard will also look at some of the fields in the
-CR object itself for some of the tests.
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
-As of `v0.8.0` scorecard also supports plugins. This allows external developers to extend the functionality of the scorecard and add additional tests.
+## Overview
 
-## Requirements
+The scorecard works by creating all resources required by CRs and the operator. 
 
-- An operator made using the `operator-sdk` or an operator that uses a config getter that supports reading from the `KUBECONFIG` environment variable (such as the `clientcmd` or `controller-runtime` config getters). This is required for the scorecard proxy to work correctly.
-- Resource manifests for installing/configuring the operator and custom resources (see the [Writing E2E Tests][writing-tests] doc for more information on the global and namespaced manifests).
-- (OLM Integration tests or Basic tests with the `olm-deployed` option set only) A CSV file for your operator.
+The scorecard will create another container in the operator deployment which is used to record calls to the API server and run a lot of the tests. These tests performe will also cover the some of the fields in the CRs objects.
 
-## Running the tests
+Since the release v0.8.0 of operator-skd, the scorecard also supports plugins which allows to extend the functionality of the scorecard and add additional tests on it. 
 
-The scorecard is configured by a config file that allows configuring both internal and external plugins as well as a few global configuration options. The configuration
-options are listed below.
+## Requirements 
 
-### Flags
+Following some requirements for the operator project which would be  checked by the scorecard. 
```

Comment:
maybe should read:  "Following are some requirements..."

Reply:
Good spot. Tks 👍 Done.